### ISSUE TITLE
BatchNorm/gamma not found in checkpoint

### ISSUE THREAD

**Issue by zuokai** [tensorflow/tensorflow#15149]
I trained a model with tensorflow1.4， and want to finetune in tensorflow 1.3，following is part of my code:
1）train code with tf1.4:
            with slim.arg_scope(inception_v3.inception_v3_arg_scope()):
          y, endpoints = inception_v3.inception_v3(x, CLASSES, True)
2）test code with master:
     with slim.arg_scope(inception_v3.inception_v3_arg_scope()):
        y, end_points = inception_v3.inception_v3(x, label_dim, False)
    with tf.name_scope('train'):
        loss = custom_function.loss_function(y, y_)
        train_op = custom_function.train_function(loss, learn_rate)
        accuracy_op = custom_function.accuracy_function(y, y_)
        output_result = custom_function.output_result(y)
        init_op = tf.group(tf.global_variables_initializer(), tf.local_variables_initializer())
        saver = tf.train.Saver()
        sess_config = tf.ConfigProto(allow_soft_placement=True, log_device_placement=False)
        sess_config.gpu_options.allow_growth = True
        sess = tf.Session(config=sess_config)
        merged_summary = tf.summary.merge_all()
        train_writer = tf.summary.FileWriter(log_dir, sess.graph)
        #saver = tf.train.import_meta_graph('model_meishi/inception-v3/graph-1205-190241.meta',clear_devices=True)
        sess.run(init_op)
        coord = tf.train.Coordinator()
        threads = tf.train.start_queue_runners(sess=sess, coord=coord)
        #saver.restore(sess, "model_meishi/model-6690")
        saver.restore(sess, "model_meishi/inception-v3/model-12042")
error is:NotFoundError (see above for traceback): Key InceptionV3/Mixed_7b/Branch_2/Conv2d_0a_1x1/BatchNorm/gamma not found in checkpoint
at first, i think the reason is slim.argscope is not the same in these two version tensorflow and 
I tried to replace the slim of tf1.4 with the slim of master，but the error still exist.
what's the problem of my code ?thx

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by zuokai** [comment]
Have I written custom code      yes
OS Platform and Distribution    centos6.5
TensorFlow installed from         anaconda
TensorFlow version                    1.4 train & 1.1 test
Bazel version                               n/a
CUDA/cuDNN version                 5.0
GPU model and memory             p40，4kernels
Exact command to reproduce     n/a

**Comment by reedwm** [comment]
Closing, since this appears to be a duplicate of tensorflow/models#2977.
